core/sql/exp/exp_eval.cpp

```diff
@@ -3003,7 +3014,7 @@ ex_expr::exp_return_type ex_expr::evalPCode(PCodeBinary* pCode32,
             if (diagsArea != atp1->getDiagsArea())
                    atp1->setDiagsArea(diagsArea);
 	    if (er == ex_expr::EXPR_ERROR) 
-	      return ex_expr::EXPR_ERROR;
+		return ex_expr::EXPR_ERROR;
```

Comment:
yes, will fix this. I use vim, but this line changed a lot in previous modification and changed the indentation, will restore it correctly.